### PR TITLE
Add list correct formatting

### PR DESCRIPTION
**Pull request by GuillaumeGomez**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -37,9 +40,53 @@ use codec::{FtpCodec, StringCodec};
 use ftp::{Answer, ResultCode};
 
 const DEFAULT_PORT: u16 = 4321;
+const MONTHS: [&'static str; 12] = ["Jan", "Feb", "Mar", "Apr", "May", "Jun",
+                                     "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"];
 
 type Writer = SplitSink<Framed<TcpStream, FtpCodec>>;
 
+cfg_if! {
```
Comment (antoyo) [inline src/main.rs:48]: What is the advantage of using that vs `#[cfg(windows)]` and `#[cfg(not(windows))]`?

Comment (GuillaumeGomez) [reply src/main.rs:48]: Not much. I just really like this macro. :)

File: src/main.rs [hunk 2]
```diff
@@ -37,9 +40,53 @@ use codec::{FtpCodec, StringCodec};
...
+    if #[cfg(windows)] {
+        fn get_file_size(meta: &Metadata) -> (time::Tm, u64) {
```
Comment (hcpl) [inline src/main.rs:50]: I think you meant `get_file_info` here.

Comment (GuillaumeGomez) [reply src/main.rs:50]: Woups, indeed!

**Pull request merged**